### ISSUE TITLE
Grabbit gives me the following error

### ISSUE THREAD

**Issue by monicapeddi021** [TWCable/grabbit#197]
I'm trying to migrate data from 1 instance to other. Both are AEM6.2 and both the servers are installed with Grabbit7.1.2 and Sun-misc-fragment-bundle 1.0.0 Softwares. So, now when I try to run grabbit.sh with the following commands:

![grabbi_commands](https://user-images.githubusercontent.com/33825013/33010899-21f80978-ce02-11e7-96d3-834830531913.png)

My config.json is as : 
![grabbi_config](https://user-images.githubusercontent.com/33825013/33010936-4ed27938-ce02-11e7-8e0f-101fb01aa452.png)

I get the following error when I run the above grabbit code: 

![grabbit2](https://user-images.githubusercontent.com/33825013/33011024-a27d6296-ce02-11e7-825a-8b0884d26b5f.png)

Any Solution with you pls?

**Comment by viveksachdeva** [comment]
Can you check comment on https://github.com/TWCable/grabbit/issues/189 and see if that helps.
"You need to supply a posix-style path to your Grabbit configuration. It won't understand windows paths."

**Comment by monicapeddi021** [comment]
Yup..I did follow that. & converted my windows path to POSIX in this way: 
Windows --> D:\Grabbit\grabbit-master\grabbit-master\config.json
Posix --> /D/Grabbit/grabbit-master/grabbit-master/config.json

So I just want to confirm if my POSIX form is correct or No? Pls help.

@viveksachdeva @jbornemann

**Comment by viveksachdeva** [comment]
The path looks correct to me.. Just occurred to me that for 6.2, you have to install one more package as mentioned in README.. Here is the package link:
https://bintray.com/twcable/aem/download_file?file_path=dependencies%2FGrabbit-Deserialization-Firewall-Configuration-1.0.zip

If you still have issues, could you check if you are seeing an error in the logs, when grabbit is installed or when you call this grabbit sync..

**Comment by monicapeddi021** [comment]
Thankyou for the response Vivek.
Yes..i did install that package and in logs it gives me: 

Access.log: 
172.22.129.38 - admin 21/Nov/2017:16:23:21 +0530 "GET /bin/receive?sling:authRequestLogin=1 HTTP/1.1" 200 32 "-" "Jakarta Commons-HttpClient/3.1"
172.22.129.38 - admin 21/Nov/2017:16:23:32 +0530 "PUT /grabbit/job HTTP/1.1" 400 127 "-" "curl/7.54.1" 

grabbit-receive.log : 
22.11.2017 15:09:19.462 *WARN* [172.22.129.38 [1511343559459] PUT /grabbit/job HTTP/1.1] com.twcable.grabbit.client.servlets.GrabbitJobServlet Bad configuration for request. relative paths provided cannot begin or end with /, ./ or \.

**Comment by monicapeddi021** [comment]
Its done. I got the output :) :)
Mistakes I did: 
1. While passing the config.json path --> no need to give the absolute path here as both grabbit.sh (which I'm running now) and config.json are under same folder. So, just giving config.json works. :+1:

2. In config.json file remove the 1st slash in path. (Didn't try with 'exclude path' concept now. But for normal paths --> it works without the slash at the beginning).

DONE. :100:
